app/population/middleware/set-regional-organisation-switcher.js

```diff
@@ -0,0 +1,13 @@
+async function setRegionalOrganisationSwitcher(req, res, next) {
```

Comment:
This is allows the "regional" details on the organisation switcher to appear.

![Screenshot from 2021-01-07 12-00-57](https://user-images.githubusercontent.com/196695/103890504-1cbd5a80-50e0-11eb-8939-e85ca77369cf.png)

Ideally this would be done everywhere for all pages, but the `/reference/location` endpoint does not accept `category` as an include, so we don't have this information on anything apart from population pages at the moment.

Reply:
As discussed above, the sub-organisational switcher feature has been removed, pending further future discussion.